src/Ryujinx.Graphics.Vulkan/VulkanInitialization.cs

```diff
@@ -41,7 +41,8 @@ public unsafe static class VulkanInitialization
             "VK_EXT_subgroup_size_control",
             "VK_NV_geometry_shader_passthrough",
             "VK_NV_viewport_array2",
-            "VK_KHR_portability_subset" // As per spec, we should enable this if present.
+            "VK_KHR_portability_subset", // As per spec, we should enable this if present.
+            "VK_EXT_depth_clip_control"
```

Comment:
```suggestion
            "VK_EXT_depth_clip_control",
            "VK_KHR_portability_subset" // As per spec, we should enable this if present.
```

I would prefer to keep VK_KHR_portability_subset as the last in the list if possible.